src/Schema/Events/MyCollection.ts

```diff
@@ -342,3 +342,21 @@ export interface SentRequestPriceEstimate {
   context_screen_owner_slug: string
   demand_index?: number
 }
+
+/**
+ * Upload another artwork
+ *
+ * This schema describes events sent to Segment from {@link ActionType.tappedUploadAnotherArtwork}
+ *
+ *  @example
+ *  ```
+ *  {
+ *    action: "tappedUploadAnotherArtwork",
+ *    context_screen_owner_type: "myCollectionArtwork"
+ *  }
+ * ```
+ */
+export interface TappedUploadAnotherArtwork {
+  action: ActionType.tappedUploadAnotherArtwork
+  context_screen_owner_type: OwnerType
+}
```

Comment:
Added a missing interface for `tappedUploadAnotherArtwork` action. @dariakoko when you free have time, please take a look if everything is ok

Reply:
thank you for quick review!